src/py/flwr/client/app.py

```diff
@@ -399,6 +399,12 @@ def start_numpy_client(
 ) -> None:
     """Start a Flower NumPyClient which connects to a gRPC server.
 
+    Warning
+    -------
+    This function is deprecated since 1.7.0. Use :code:`flwr.client.start_client`
+    instead and first convert your :code:`NumPyClient` to type :code:`flwr.client.Client` by
+    exectuing its :code:`to_client()` method.
```

Comment:
```suggestion
    This function is deprecated since 1.7.0. Use :code:`flwr.client.start_client`
    instead and first convert your :code:`NumPyClient` to type
    :code:`flwr.client.Client` by executing its :code:`to_client()` method.
```